### DISCUSSION TITLE
docs/comments

### DISCUSSION THREAD

**Discussion by giscus** [flowershow/flowershow#615] [Announcements]
# docs/comments

Get more feedback or simply increase visitor engagement by easily adding page comments to your Flowershow website or blog.

https://flowershow.app/docs/comments

**Comment by Love4yzp** [comment]
Hi, where is the `.flowershow` folder, I didn't see it. However, I see a folder `_flowershow`.

**Reply by Love4yzp** [reply]
Yes, I set them in vercel manually, and it correctly run.
